City-of-Pori/pori-kada
# PORI-402 Added telephone exchange number to phonebook

To test:

1. drush fra -y && drush cc all
2. Go to phonebook page https://local.pori.fi/puhelinluettelo
3. Make sure you can see the text under the page title "Telephone exchange of the City of Pori: (02) 621 1100
kirjaamo@pori.fi"
4. Go to https://local.pori.fi/en/admin/config/regional/translate/translate and search for Telephone exchange
5. Translate the English version to Puhelinvaihde:
6. Go to https://local.pori.fi/puhelinluettelo and make the translation works. Now try the English version of search as well https://local.pori.fi/en/phone-book and make sure it works too.
